pkg/controller/storagecluster/initialization_reconciler_test.go

```diff
@@ -352,6 +395,59 @@ func assertExpectedResources(t assert.TestingT, reconciler ReconcileStorageClust
 	assert.Equal(t, len(expectedCos[0].OwnerReferences), 1)
 }
 
+func TestEnsureExternalStorageClusterResources(t *testing.T) {
+	jsonBlob, _ := json.Marshal(ExternalResources)
+
+	request := reconcile.Request{
+		NamespacedName: types.NamespacedName{
+			Name:      "ocsinit",
+			Namespace: "",
+		},
+	}
+	cr := &corev1.Secret{
```

Comment:
This should not be `cr`, name it something like `externalSecret`. Don't forget to call `reconciler.client.Create()` as well.

Reply:
addressed